app/controllers/doc_controller.rb

```diff
@@ -97,7 +95,7 @@ def set_book
   def set_doc_file
     file  = params[:file]
     if DocFile.exists?(name: file)
-      @doc_file = DocFile.with_includes.where(name: file).limit(1).first
```

Comment:
you removed this line @peff, where the problem came from. I guess we are now using other route/method to get the contents

Reply:
Did I? It still seems to be there: https://github.com/git/git-scm.com/blob/46247ff2a867e6afc9365fa61d9ec05585b680bf/app/controllers/doc_controller.rb#L57